code/pizzabot.js

```diff
@@ -1,7 +1,42 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaMenu = [vegetarian, hawaiian, pepperoni];
+const pizzaPrice = 80;
 
-const pizzaPrice = 80
+document.getElementById("message").innerHTML = `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`;
 
-//Put your Javscript code here:
+const checkOrderName = (pizza) => {
+  for (let i = 0; i < pizzaMenu.length; i++) {
+    if (pizzaMenu[i].toLowerCase() === pizza.toLowerCase()) {
+      return true;
+    }
+  }
+  return false;
+};
+
+const totalCost = (quantity) => {
+  return pizzaPrice * +quantity;
+};
+
+const cookingTime = (quantity) => {
+  if (+quantity <= 2) {
```

Comment:
How come you are adding `+` before?

Reply:
> How come you are adding `+` before?

To convert string to a number. We previously used `parseFloat` or `parseInt` but then we learned from last weeks codealong that we could use the `+`operator  instead.

Reply:
That's true! But the `quantity` is never a string as I can see? :)